packages/select/src/Select.js

```diff
@@ -90,6 +92,15 @@ const Select = ({
     _cacheUniq = [_cacheUniq];
   }
 
+  attributes.placeholder = (
+    <>
+      {attributes.placeholder || 'Select...'}
+      <span className="sr-only">
+        {(touched && hasError) || null} {helpMessage || null}
```

Comment:
I'm not sure what this part `{(touched && hasError) || null}` is for? Can you elaborate?

Reply:
Ohhh, I thought it was a boolean based on the name. Based on the types it should always be a string so I think we're good.

Reply:
can we rename the variable from `hasError` to something like `fieldError` for clarity?